rgw/v2/tests/s3_swift/test_rgw_account_management.py

```diff
@@ -26,10 +26,11 @@
 from v2.lib.exceptions import RGWBaseException, TestExecError
```

Comment:
can we please update the conf file detail's in script

Reply:
@anrao19 Ack and done.